pkg/api/pod/warnings.go

```diff
@@ -172,6 +172,9 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		if v.PhotonPersistentDisk != nil {
 			warnings = append(warnings, fmt.Sprintf("%s: non-functional in v1.16+", fieldPath.Child("spec", "volumes").Index(i).Child("photonPersistentDisk")))
```

Comment:
pre-existing, but let's add the deprecated version here as well

```suggestion
			warnings = append(warnings, fmt.Sprintf("%s: deprecated in v1.11, non-functional in v1.16+", fieldPath.Child("spec", "volumes").Index(i).Child("photonPersistentDisk")))
```

Reply:
Sure, make sense. Do we want to also add warnings to all other deprecated plugins here?

Reply:
the ones that have in-tree → csi translations shouldn't get warnings, right, since using the API field is still valid, it's just serviced by a CSI plugin

for any that are purely deprecated, yes, add warnings with the version they were deprecated in

Reply:
Done.